### PR TITLE
Proposal to donate Kubernetes Event-driven Autoscaling (KEDA) as Sandbox project

### PR DESCRIPTION
**Pull request by tomkerkhove**
Thanks for considering us for recomendation to be donated to CNCF as a sandbox project.

We've filled in the template which should contain all the information that is required.
Feel free to ask any questions or more information in case there is a need for it

CC @jeffhollan @anirudhgarg

_Relates to https://github.com/cncf/toc/issues/335_
_Relates to kedacore/keda#501_

### CODE REVIEW

**Comment by jeffhollan** [comment]
Thanks all - @raravena80 let us know if you have any questions for us. Thanks for the time today

File: recommendations/sandbox/keda.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+## Background
+
+**_Link to TOC PR_**
+
+https://github.com/cncf/toc/issues/335
+
+**_Link to Presentation_**
+
+Recording: Feb 20 SIG-Runtime meeting
```
Comment (raravena80) [inline recommendations/sandbox/keda.md]: Can we update the link here as soon as the presentation gets posted on the CNCF channel: https://www.youtube.com/channel/UCvqbFHwN-nwalWPjPUKpvTA? 

cc: @amye 😄

File: recommendations/sandbox/keda.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+
+## Alignment with other CNCF projects
+
+KEDA augments and enhances many CNCF projects by adding event driven scale, including:
+
+- Kubernetes itself (helping drive features back into things like the HPA / Cluster autoscaler)
+- Virtual Kubelet
+- NATS
+- Promethues
```
Comment (raravena80) [inline recommendations/sandbox/keda.md]: Typo: Prometheus

File: recommendations/sandbox/keda.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+![KEDA Architecture](https://i.imgur.com/lBVkWsH.png)
+
+# Formal Requirements
+
+Sponsors: TBD
+
+# CNCF IP Policy
+
+	cloud.google.com/go (Apache License 2.0)
```
Comment (raravena80) [inline recommendations/sandbox/keda.md]: Nitpick, indentation for all these. Can be either:

- cloud.google.com/go (Apache License 2.0)
...
or 

cloud.google.com/go (Apache License 2.0)
...

File: recommendations/sandbox/keda.md (hunk 2 above)
Comment (tomkerkhove) [reply recommendations/sandbox/keda.md]: Done

File: recommendations/sandbox/keda.md (hunk 1 above)
Comment (tomkerkhove) [reply recommendations/sandbox/keda.md]: We will! Feel free to let us know when you're done!

File: recommendations/sandbox/keda.md (hunk 3 above)
Comment (tomkerkhove) [reply recommendations/sandbox/keda.md]: Done

File: recommendations/sandbox/keda.md (hunk 1 above)
Comment (tomkerkhove) [reply recommendations/sandbox/keda.md]: Any update on this @amye?

**Comment by jeffhollan** [comment]
@raravena80 is it cool if we just edit this and keep only link to deck? Not sure if any changes or discussion needed before moving to next stage and seems only to be held up by video link which I wanted to check to see if we could progress without :)

File: recommendations/sandbox/keda.md (hunk 1 above)
Comment (tomkerkhove) [reply recommendations/sandbox/keda.md]: Is this a blocker for merging @raravena80 or can we proceed and add it after the fact?

**Comment by raravena80** [comment]
@jeffhollan @tomkerkhove at the moment having the presentation link is part of the Sandbox template, some TOC members may prefer that format over browsing through the slides.

Merging this PR will constitute the recommendation from SIG-Runtime and then as per the [current Sandbox process](https://github.com/cncf/toc/blob/master/process/project_proposals.adoc#sandbox-process) the following steps are TOC review and getting 3 TOC sponsors (which means entry to Sandbox). Note that the sponsors need to be current [members of the TOC](https://github.com/cncf/toc#members).

Thanks!

**Comment by tomkerkhove** [comment]
Thanks for the info @raravena80!

**Comment by tomkerkhove** [comment]
We should be good to go now, thanks for re-reviewing @raravena80!

**Comment by tomkerkhove** [comment]
Thank you so much @raravena80!

**Pull request merged**